.travis.yml

```diff
@@ -3,14 +3,12 @@ os:
 - osx
 osx_image: xcode8.3
 rvm:
-- 2.5.0
-- 2.4.3
-- 2.3.6
-- 2.2.9
-- 2.1.10
+- 2.5.1
```

Comment:
The build scripts will need to be updated to use these same versions, since they also use `rvm-exec`.